### PR TITLE
remote: grpcs (grpc+tls) is used by default for remote URLs

### PR DESCRIPTION
**Pull request by ishikhman**
RELNOTES: --remote_executor, --remote_cache or --bes_backend=someurl.com would be treated as grpcs://someurl.com, if the --incompatible_tls_enabled_removed flag enabled. See https://github.com/bazelbuild/bazel/issues/8061 for details.

### CODE REVIEW

File: src/main/java/com/google/devtools/build/lib/authandtls/GoogleAuthUtils.java [hunk 1]
```diff
@@ -91,6 +91,10 @@ private static boolean isTlsEnabled(String target, AuthAndTLSOptions options) {
     if (options.incompatibleTlsEnabledRemoved && options.tlsEnabled) {
       throw new IllegalArgumentException("flag --tls_enabled was not found");
     }
+    if (options.incompatibleTlsEnabledRemoved) {
+      // 'grpcs://' or empty prefix => TLS-enabled
```
Comment (buchgr) [inline src/main/java/com/google/devtools/build/lib/authandtls/GoogleAuthUtils.java]: Maybe expand the comment on empty prefix a bit?

**Comment by buchgr** [comment]
That's great! Thanks!

File: src/main/java/com/google/devtools/build/lib/authandtls/GoogleAuthUtils.java (hunk 1 above)
Comment (ishikhman) [reply src/main/java/com/google/devtools/build/lib/authandtls/GoogleAuthUtils.java]: done

**Comment by buchgr** [comment]
LGTM

**Pull request closed without merging**